queries/gularen/injections.scm

```diff
@@ -0,0 +1,24 @@
+(code_block_lang
+  (code_lang) @injection.language
+  (code_content) @injection.content)
+
+(code_inline_lang
+  (code_inline_content) @injection.content
+  (label) @injection.language)
+
+; common aliases
+
+(code_block_lang
+  (code_lang) @_language
+  (#any-of? @_language "js")
+  (code_content) @javascript)
+
+(code_block_lang
+  (code_lang) @_language
+  (#any-of? @_language "py")
+  (code_content) @python)
+
+(code_block_lang
+  (code_lang) @_language
+  (#any-of? @_language "sh" "zsh")
+  (code_content) @bash)
```

Comment:
Please update this to the same style as the ones above

Reply:
My apologize I do not understand the query syntax, I just copy pasted from @ObserverOfTime on this issue #3704

Can you give me an example of what you mean?

Reply:
Simply put, you are not allowed to do that kind of injection anymore. Please change it to `@injection.content (#set! injection.language "bash")` and the likes.

Reply:
I have changed it